mvn clean install fails

http://devel.summitsystemsinc.net/artifactory/repo/oculus/aperture/aperture/1.0.9-SNAPSHOT/aperture-1.0.9-SNAPSHOT.pom 

Requests authentication but is one of the POM files that Maven tries to download during a clean install.  Please advise.

mvn clean install
[INFO] Scanning for projects...
Downloading: http://devel.summitsystemsinc.net/artifactory/repo/oculus/aperture/aperture/1.0.9-SNAPSHOT/maven-metadata.xml
[WARNING] Could not transfer metadata oculus.aperture:aperture:1.0.9-SNAPSHOT/maven-metadata.xml from/to codelutin (http://devel.summitsystemsinc.net/artifactory/repo/): Not authorized , ReasonPhrase:Unauthorized.
Downloading: http://devel.summitsystemsinc.net/artifactory/repo/oculus/aperture/aperture/1.0.9-SNAPSHOT/aperture-1.0.9-SNAPSHOT.pom
[ERROR] The build could not read 3 projects -> [Help 1]
[ERROR]  
[ERROR]   The project oculus.aperture.service:aperture-geo:1.0.9-SNAPSHOT (/root/src/aperturejs/aperture-geo/pom.xml) has 1 error
[ERROR]     Non-resolvable parent POM: Could not find artifact oculus.aperture:aperture:pom:1.0.9-SNAPSHOT and 'parent.relativePath' points at wrong local POM @ line 6, column 10 -> [Help 2]
[ERROR]  
[ERROR]   The project oculus.aperture:aperture-graph:1.0.9-SNAPSHOT (/root/src/aperturejs/aperture-graph/pom.xml) has 1 error
[ERROR]     Non-resolvable parent POM: Could not transfer artifact oculus.aperture:aperture:pom:1.0.9-SNAPSHOT from/to codelutin (http://devel.summitsystemsinc.net/artifactory/repo/): Not authorized , ReasonPhrase:Unauthorized. and 'parent.relativePath' points at wrong local POM @ line 6, column 10 -> [Help 2]
[ERROR]  
[ERROR]   The project oculus.aperture:aperture-examples:1.0.9-SNAPSHOT (/root/src/aperturejs/aperture-examples/pom.xml) has 1 error
[ERROR]     Non-resolvable parent POM: Could not find artifact oculus.aperture:aperture:pom:1.0.9-SNAPSHOT and 'parent.relativePath' points at wrong local POM @ line 3, column 10 -> [Help 2]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/ProjectBuildingException
[ERROR] [Help 2] http://cwiki.apache.org/confluence/display/MAVEN/UnresolvableModelException

--- a/aperture-common/pom.xml
+++ b/aperture-common/pom.xml
@@ -7,7 +7,6 @@
 		<artifactId>aperture</artifactId>
 		<groupId>oculus.aperture</groupId>
 		<version>1.1.0-SNAPSHOT</version>
-		<relativePath>../pom.xml</relativePath>
 	</parent>
 
 	<artifactId>aperture-common</artifactId>

--- a/aperture-examples/pom.xml
+++ b/aperture-examples/pom.xml
@@ -3,7 +3,7 @@
 	<parent>
 		<artifactId>aperture</artifactId>
 		<groupId>oculus.aperture</groupId>
-		<version>1.0.9-SNAPSHOT</version>
+		<version>1.1.0-SNAPSHOT</version>
 	</parent>
 
 	<artifactId>aperture-examples</artifactId>

--- a/aperture-geo/pom.xml
+++ b/aperture-geo/pom.xml
@@ -6,7 +6,7 @@
 	<parent>
 		<artifactId>aperture</artifactId>
 		<groupId>oculus.aperture</groupId>
-		<version>1.0.9-SNAPSHOT</version>
+		<version>1.1.0-SNAPSHOT</version>
 	</parent>
 
 	<groupId>oculus.aperture.service</groupId>

--- a/aperture-graph/pom.xml
+++ b/aperture-graph/pom.xml
@@ -6,8 +6,7 @@
 	<parent>
 		<artifactId>aperture</artifactId>
 		<groupId>oculus.aperture</groupId>
-		<version>1.0.9-SNAPSHOT</version>
-		<relativePath>../pom.xml</relativePath>
+		<version>1.1.0-SNAPSHOT</version>
 	</parent>
 
 	<artifactId>aperture-graph</artifactId>

--- a/aperture-layout-yworks/pom.xml
+++ b/aperture-layout-yworks/pom.xml
@@ -6,8 +6,7 @@
 	<parent>
 		<artifactId>aperture</artifactId>
 		<groupId>oculus.aperture</groupId>
-		<version>1.0.9-SNAPSHOT</version>
-		<relativePath>../pom.xml</relativePath>
+		<version>1.1.0-SNAPSHOT</version>
 	</parent>
 
 	<groupId>oculus.aperture.service</groupId>

